cmd/mistryd/worker.go

```diff
@@ -238,6 +221,35 @@ func (s *Server) Work(ctx context.Context, j *Job) (buildInfo *types.BuildInfo,
 		return
 	}
 
+	client, err := docker.NewEnvClient()
+	if err != nil {
+		err = workErr("could not create docker client", err)
+		return
+	}
+
+	err = j.BuildImage(ctx, s.cfg.UID, client, out)
+	if err != nil {
+		return
```

Comment:
Just noticed that we do not set any errors here. We should fix this at this PR.

Reply:
done